config/vendor-api-v1.yml

```diff
@@ -679,6 +689,23 @@ components:
           - Completion of professional skills test
         course:
           "$ref": "#/components/schemas/Course"
+        offer_made_at:
+          type: string
+          format: date-time
+          description: Date and time the offer was made
+          example: 2019-09-18T15:33:49.216Z
+        accepted_at:
```

Comment:
Is it worth prefixing this and `declined_at` with `offer_` for consistency?

Reply:
Yeah, I wondered about that! I think you're right — will update